### PR TITLE
add minikube integration tests to github actions

### PR DESCRIPTION
**Pull request by medyagh**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -165,6 +165,10 @@ clean:
 kind-cluster:
 	kind get clusters | grep -q kind || kind create cluster
 
+.PHONY: minikube-cluster
+minikube-cluster:
+	KUBECONFIG=/tmp/skaffold_test_kubeconfig minikube start --vm-driver=docker
```
Comment (balopat) [inline Makefile]: `--container-runtime=docker`?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: install minikube
+      run : |
+       curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 
+       sudo install minikube-linux-amd64 /usr/local/bin/minikube
+    - name: run integration test
+      run: |
+        make integration-in-minikube
+  integration_minikube_ubuntu_18_04:
```
Comment (balopat) [inline .github/workflows/main.yml]: duplicated?

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on: [pull_request]
+
+jobs:
+  integration_minikube_ubuntu_16_04:
```
Comment (balopat) [inline .github/workflows/main.yml]: do we really need to test on multiple ubuntu's?

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
+name: CI
```
Comment (balopat) [inline .github/workflows/main.yml]: why github actions versus TravisCI?

Comment (nkubala) [reply .github/workflows/main.yml]: i'm not sure we want to migrate any of our other integration tests over to github actions at this time, but i'd be curious to see if github actions performs any better than travis.

**Comment by tstromberg** [comment]
Closing this draft as it's getting stale

**Pull request closed without merging**